.github/workflows/pull-request.yml

```
@@ -561,3 +561,26 @@ jobs:
+          node-version: '22.x'
+
+      - name: Cache npm modules
+        uses: actions/cache@v4
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json', 'package.json') }}
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Check for Unused and Missing Dependencies
+        run: npm run check-dependencies
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)